reVX/wind_setbacks/wind_setbacks.py

```diff
@@ -0,0 +1,1386 @@
+            blade tip height which is used to determine setback distance
+        regs_fpath : str | None, optional
+            Path to wind regulations .csv file, if None create global
+            setbacks, by default None
+        multiplier : int | float | str | None, optional
+            setback multiplier to use if wind regulations are not supplied,
+            if str, must one of {'high': 3, 'moderate': 1.1},
+            by default None
+        chunks : tuple, optional
+            Chunk size to use for setback layers, if None use default chunk
+            size in excl_h5, by default (128, 128)
+        max_workers : int, optional
+            Number of workers to use for setback computation, if 1 run in
+            serial, if > 1 run in parallel with that many workers, if None
+            run in parallel on all available cores, by default None
+        description : str, optional
+            Description of exclusion layer (set as an attribute),
+            by default None
+        replace : bool, optional
+            Flag to replace existing layer data with arr if layer already
+            exists in the exlcusion .h5 file, by default False
+        """
+        setbacks = cls(excl_h5, hub_height, rotor_diameter,
+                       regs_fpath=regs_fpath, multiplier=multiplier,
+                       hsds=False, chunks=chunks)
+        setbacks.compute_setbacks(roads_dir, layer=layer_name,
+                                  max_workers=max_workers,
+                                  description=description,
+                                  replace=replace)
+
+
+class TransmissionWindSetbacks(BaseWindSetbacks):
+    """
+    Transmission Wind setbacks
+    """
+    @staticmethod
+    def _parse_regs(regs_fpath):
+        """
+        Parse wind regulations, reduce table to just transmission
+
+        Parameters
+        ----------
+        regs_fpath : str
+            Path to wind regulations .csv file
+
+        Returns
+        -------
+        regs : pandas.DataFrame
+            Wind regulations table
+        """
+        regs = BaseWindSetbacks._parse_regs(regs_fpath)
+
+        mask = regs['Feature Type'] == 'Transmission'
+        regs = regs.loc[mask]
+
+        return regs
+
+    @staticmethod
+    def _parse_features(transmission_fpath, crs):
+        """
+        Load transmission shape file, convert to exclusions coordinate system
+
+        Parameters
+        ----------
+        transmission_fpath : str
+            Path to transmission shape file
+        crs : str
+            Coordinate reference system to convert structures geometries into
+
+        Returns
+        -------
+        trans : geopandas.GeoDataFrame.sindex
+            Geometries for transmission features, in exclusion coordinate
+            system
+        """
+        trans = gpd.read_file(transmission_fpath)
+        mask = (~(trans['Proposed'] == 'Proposed')
+                & ~(trans['Undergroun'] == 'T'))
```

Comment:
Undergroun a typo? (missing the d)

Also this is a super weird mask. Why not use `!=` instead of `~==`?